packages/components/buttons/primary-button/src/primary-button.tsx

```diff
@@ -61,6 +61,7 @@ export type TPrimaryButtonProps<
   ) => void;
   /**
    * Indicates the size of the button.
+   * @deprecated `small` value is deprecated. Use `medium` instead.
```

Comment:
I would maybe not add the `@deprecated` keyword as it's like we are deprecating the `size` property which we're not:
```suggestion
   * Indicates the size of the button.
   * <br />
   * `small` value has been deprecated. Please use `medium` value instead as a replacement.
```

Reply:
Done here - [c6c0c68](https://github.com/commercetools/ui-kit/pull/2455/commits/c6c0c682e028d0edd0a3a2034bb189c4e3dd50fe) ✅